includes/payment-tokens/class-wc-payment-token-cc.php

```diff
@@ -0,0 +1,115 @@
+<?php
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit; // Exit if accessed directly
+}
+
+/**
+ * WooCommerce Credit Card Payment Token
+ *
+ * Representation of a payment token for credit cards
+ * Extends from WC_Payment_Token_eCheck since both types contain last4 digits.
+ *
+ * @class 		WC_Payment_Token_CC
+ * @since		2.6.0
+ * @category 	PaymentTokens
+ * @package 	WooCommerce/PaymentTokens
+ * @author		WooThemes
+ */
+class WC_Payment_Token_CC extends WC_Payment_Token_eCheck {
+
+ 	/**
+	 * Validate credit card payment tokens
+	 *
+	 * These fields are required by all credit card payment tokens:
+	 * expiry_month  - string Expiration date (MM) for the card
+	 * expiry_year   - string Expiration date (YYYY) for the card
+	 * last4         - string Last 4 digits of the card
+	 * card_type     - string Card type (visa, mastercard, etc)
+	 *
+	 * @since 2.6.0
+	 * @param array $args Data to validate
+	 * @return boolean    True if the passed data is valid
+	 */
+	public static function validate( $args ) {
```

Comment:
The intention was so devs could validate arguments early without the need to call update/insert. And in the case of inserting a token with `create`, we don't have an instance of the token object yet when we validate.